ts/Core/Series/Series.ts

```diff
@@ -4375,6 +4375,8 @@ class Series {
 
         series.initialType = initialType;
         chart.linkSeries(); // Links are lost in series.remove (#3028)
+        // setData isn't called with dataSorting & dataGrouping enabled (#19715)
```

Comment:
The comment is misleading - what does it means that it isn't called? Describe what happens and why we do it, not what failed prior to the fix.